ckanext/nextgeoss/public/nextgeoss.css

```diff
@@ -822,7 +822,6 @@ section.dataset-map-section {
 
 .search-form .search-input.search-giant button {
   left: 20px;
-  margin-top: -12px;
```

Comment:
@kmbn this was added for aligning the search icon using Font Awesome. Maybe I did something wrong here, but let's test it out and make changes if needed